tempesta_fw/http.h

```diff
@@ -453,10 +455,14 @@ typedef struct {
 	time_t			date;
 	time_t			last_modified;
 	unsigned long		jrxtstamp;
+	TfwHttpMsg		*req;
```

Comment:
An alternative is moving this to `TfwHttpMsg{}/TFW_HTTP_MSG_COMMON`, and also move `TfwHttpReq{}->resp` there. Then, there it could be either a union of `{ req, resp }`, or simply a single member, say, `->pair`.

Reply:
I see the point here, this will also make unneeded all that `BOUND_REQ` macro.